team-config.yml

```diff
@@ -102,10 +102,10 @@ probate:
     contact_channel: "#probate"
     build_notices_channel: "#probate-jenkins"
 bulk-scan:
-  team: "Software Engineering"
+  team: "Bulk Scanning and Printing"
   namespace: "rpe"
```

Comment:
Do u want to stick to rpe namespace?

Reply:
it'll break all our current build will it not?

Reply:
yes, just needs some one-off manual intervention